ambry-router/src/test/java/com/github/ambry/router/PutOperationTest.java

```diff
@@ -969,4 +974,62 @@ public void compressChunk()
     ByteBuf buf = (ByteBuf) FieldUtils.readField(putChunk, "buf", true);
     buf.release();
   }
+
+  /**
+   * Ensure that if any of the requests associated with the buffer of a PutChunk is not completely read out even
+   * after the associated chunk is complete, the buffer is not reused even though the PutChunk is reused.
+   */
+  @Test
+  public void testRetry() throws Exception {
```

Comment:
could you please change the function to something like testRetryOnSimpleOperationTrackerConstructorFailure?